rasa/core/policies/ensemble.py

```diff
@@ -375,47 +401,100 @@ def probabilities_using_best_policy(
                     domain.index_for_action(tracker.events[-1].action_name)
                 ] = 0.0
 
-            confidence = np.max(probabilities)
-            if (confidence, p.priority) > (max_confidence, best_policy_priority):
-                max_confidence = confidence
-                result = probabilities
-                best_policy_name = "policy_{}_{}".format(i, type(p).__name__)
-                best_policy_priority = p.priority
+            if isinstance(p, FormPolicy):
+                # get form predictions separately
+                form_probabilities = probabilities
+                form_policy_index = i
+                form_policy_priority = p.priority
+            else:
+                # pick the best policy
+                confidence = max(probabilities)
+                if (confidence, p.priority) > (max_confidence, best_policy_priority):
+                    max_confidence = confidence
+                    result = probabilities
+                    best_policy_name = f"policy_{i}_{type(p).__name__}"
+                    best_policy_priority = p.priority
```

Comment:
optional: extract it a function `pick_best_policy(...)` instead of adding a comment to the code.